src/LodeRunner.Core/Models/LoadResult.cs

```diff
@@ -1,12 +1,73 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using LodeRunner.Core.SchemaFilters;
+using Swashbuckle.AspNetCore.Annotations;
+
 namespace LodeRunner.Core.Models
 {
     /// <summary>
     /// LoadResult model.
     /// </summary>
-    public class LoadResult : BaseEntityModel
+    [SwaggerSchemaFilter(typeof(LoadResultSchemaFilter))]
+    public class LoadResult
     {
+        /// <summary>
+        /// Gets or sets the LoadClient.
+        /// </summary>
+        /// <value>
+        /// LoadClient.
+        /// </value>
+        [Required]
+        [Description("LoadClient load test ran on.")]
+        public virtual LoadClient LoadClient { get; set; }
```

Comment:
Is there any reason to make this "virtual"?

Reply:
@alfredoihernandez

Reply:
removed